libpod/container_internal.go

```diff
@@ -1314,16 +1314,24 @@ func (c *Container) stop(timeout uint) error {
 	// state has changed.  If so, we should return immediately and log a
 	// warning.
 	if c.state.State != define.ContainerStateStopping {
-		logrus.Warnf(
-			"Container %q state changed from %q to %q while waiting for it to be stopped: discontinuing stop procedure as another process interfered",
-			c.ID(), define.ContainerStateStopping, c.state.State,
-		)
+		if c.state.State != define.ContainerStateExited {
```

Comment:
just combine both if blocks into one

Reply:
That would break the logic.  If the state is "stopping" we want to return.  Though we only warn if the state is anything other than "exited".